flang/lib/Lower/Bridge.cpp

```diff
@@ -231,6 +231,13 @@ class FirConverter : public Fortran::lower::AbstractConverter {
   // AbstractConverter overrides
   //===--------------------------------------------------------------------===//
 
+  /// Find the symbol in the local map or return null.
+  mlir::Value lookupSymbol(const Fortran::semantics::Symbol &sym) {
```

Comment:
This was mindlessly moving code from one place to another without considering the details.  Do you have a suggested update?

Reply:
Can we call it `getMlirValueRepresentingTheSymbolAddress()` or something (preferably much shorter)?

Reply:
`lookupSymbol` has 16 or so hits in Bridge.cpp plus the one new use now in IO.cpp.  Since the function's future is looking doubtful, I think I prefer to do a hit and run use, and leave more fundamental cleanup to someone with a better focus on the problem than I currently have.

Reply:
Ok. I'll change it later.